docs/openapi.yaml

```diff
@@ -213,6 +213,56 @@ paths:
               example:
                 $ref: ./api/transaction/get-mempool-transactions.example.json
 
+  /extended/v1/tx/tx_list_details:
```

Comment:
@rafaelcr @agraebe thoughts on a possible more REST-compliant and readable endpoint name?

Reply:
this is essentially a filter on all txs by a property (ids in this case), right? If so, how about one of these:
-  /extended/v1/tx?id=x,y,z
-  /extended/v1/tx/filter?id=x,y,z

Reply:
The purpose of this endpoint is not to filter transactions. It is designed to return the details for a list of transaction ids.

Reply:
Yep, @zone117x `/multiple` sounds good. I'll change the endpoint to that.